CHANGELOG.md

```diff
@@ -8,50 +8,64 @@
-* *Single-property components*. A component can define itself as consisting of only one property by specifying a type and/or a default value in the schema. (d35e56e)
-* *Asset management system* that blocks scene render
-* *Play/pause* methods on entities and play/pause handlers on components. (9238861)
-* *Tick* method on components to register a function called on each scene tick. (#823)
-* Support for loading *.OBJ* and .MTL assets. (#788)
-* *Texture caching* for better performance when reusing textures. (#1116)
-* Components can be attached to primitives (e.g., `<a-sphere>`). (#883)
-* Introduce *systems* API (unstable) to provide global scope and services for components. (#924)
-* Entities, including the scene, wait for their children to load before emitting the `loaded` event. (a8a4f06)
-* Entities emit `child-attach` when children are attached.
-* Lot of `<a-scene>` logic moved to configurable components. (#776)
-* Support for *multiple cameras* in a scene and switching between them. (#745)
-* Added more events for scene VR mode, material component, model components, sound component.
+- Introduce *shaders* to extend the material component and to register custom GLSL shaders. (#861)
+- Component *property types*. Property types define how a component property is parsed and stringified. Custom property types can be registered or defined inline with the property in the schema. Built-in property types include `array`, `boolean`, `color`, `int`, `number`, `selector`, `selectorAll`, `string`, `vec2`, `vec3`, `vec4`. (d35e56e)
+- *Single-property components*. A component can define itself as consisting of only one property by specifying a type and/or a default value in the schema. (d35e56e)
+- *Asset management system* that blocks scene render
+- *Play/pause* methods on entities and play/pause handlers on components. (9238861)
+- *Tick* method on components to register a function called on each scene tick. (#823)
+- Support for loading *.OBJ* and .MTL assets. (#788)
+- *Texture caching* for better performance when reusing textures. (#1116)
+- Components can be attached to primitives (e.g., `<a-sphere>`). (#883)
+- Introduce *systems* API (unstable) to provide global scope and services for components. They can be registered wtih `AFRAME.registerSystem`. (#924)
+- Entities, including the scene, wait for their children to load before emitting the `loaded` event. (a8a4f06)
+- Entities emit `child-attach` when children are attached.
+- Most `<a-scene>` logic moved to configurable components and systems. (#776)
+- Support for *multiple cameras* in a scene and switching between them. (#745)
+- Added more events for scene VR mode, material component, model components, sound component.
```

Comment:
do you want to list the events - or too many to list? maybe link to docs? but probably not necessary.